pkg/opa/opa_calls.go

```diff
@@ -96,12 +96,18 @@ func DeleteCheck(check models.CustomCheckModel, org, token, hostName string) err
 type PutCheckRequest struct {
 	Rego, Description string
 	Disabled          *bool
+	RegoVersion       string
 }
 
 // PutCheck upserts an OPA Check to Fairwinds Insights
-func PutCheck(check models.CustomCheckModel, org, token, hostName string) error {
+func PutCheck(check models.CustomCheckModel, org, token, hostName string, pushRegoVersion string) error {
 	url := fmt.Sprintf(opaPutCheckURLFormat, hostName, org, check.CheckName, check.Version)
 	body := PutCheckRequest{Rego: check.Rego, Description: check.Description, Disabled: check.Disabled}
+	if pushRegoVersion != "" {
```

Comment:
[opt] I assume this is never empty, due to the default specified via `cobra`
would be probably wise to validate this parameter for either `v0` or `v1` (or leave the BE to do so)